example/src/components/examples/Alerts.js

```diff
@@ -6,32 +6,26 @@ const Alerts = () => (<>
 
   <p>Information:</p>
   <Alert
-    type='info'
-    analyticsString='object.action.event'
     className='extra-class'
     data-extra-attribute
     dismissible={false}
-    idString='alert-info-1'
+    type='info'
```

Comment:
thought I updated this with the commit I made for the close icon, but I guess I didn't 😛

Reply:
You did, but I alphabetized the props.